Integration/cptIMSCobraExport_bas.bas

```diff
@@ -5865,12 +5868,18 @@ End Sub
 Private Function SetDirectory(ByVal ProjName As String) As String
     Dim newDir As String
     Dim pathDesktop As String
+    Dim fs As FileSystemObject 'v3.4.1
 
     pathDesktop = CreateObject("WScript.Shell").SpecialFolders("Desktop")
     newDir = pathDesktop & "\" & RemoveIllegalCharacters(ProjName) & "_" & Format(Now, "YYYYMMDD HHMMSS")
 
+    If Len(newDir) > 220 Then 'v3.4.1
+        newDir = "\\?\" & newDir
+    End If
+
     MkDir newDir
-    SetDirectory = newDir
+    Set fs = CreateObject("Scripting.FileSystemObject") 'v3.4.1
+    SetDirectory = fs.GetFolder(newDir).ShortPath 'v3.4.1
     Exit Function

```

Comment:
`Set fs = Nothing` before `Exit Function`?

Reply:
I meant to upload a version that didn't use the FS object. Let me try this again.